2021-offseason/src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +14,21 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class kDrivetrain{
+        public static final int frontLeft_ID = 18;
```

Comment:
nit: style: names should be capitalized to follow constant naming, e.g. `FRONT_LEFT_ID`